app/views/welcome/index.html.erb

```diff
@@ -4,6 +4,13 @@
 <div class="col-md-6 col-md-offset-3 text-center">
     <h3>Future Home Of the Fall Gambit Chess App!</h3>
     <br />
-    <%= render "available_games" %>
+    <% if @games.present? %>
```

Comment:
Isn't this code (starting at line 7 and going to line 14) repeating what is currently part of an if-then-else in the _available_games partial above?